spyder/widgets/about.py

```diff
@@ -105,70 +161,136 @@ def __init__(self, parent):
             See the <a href="{github_url}/blob/master/NOTICE.txt">NOTICE</a>
             file for full legal information.
             </p>
             </div>
             """).format(
-                spyder_ver=versions['spyder'],
-                revision=revlink,
-                website_url=__website_url__,
                 github_url=__project_url__,
-                trouble_url=__trouble_url__,
-                forum_url=__forum_url__,
-                python_ver=versions['python'],
-                bitness=versions['bitness'],
-                qt_ver=versions['qt'],
-                qt_api=versions['qt_api'],
-                qt_api_ver=versions['qt_api_ver'],
-                os_name=versions['system'],
-                os_ver=versions['release'],
                 font_family=font_family,
                 font_size=font_size,
             )
         )
-        self.label.setWordWrap(True)
-        self.label.setAlignment(Qt.AlignTop)
-        self.label.setOpenExternalLinks(True)
-        self.label.setTextInteractionFlags(Qt.TextBrowserInteraction)
-        self.label.setFixedWidth(350)
+        self.label_overview.setWordWrap(True)
+        self.label_overview.setAlignment(Qt.AlignTop)
+        self.label_overview.setOpenExternalLinks(True)
+        self.label_overview.setTextInteractionFlags(Qt.TextBrowserInteraction)
+        self.label_overview.setContentsMargins(15, 0, 25, 0)
+
+        self.label_community.setWordWrap(True)
+        self.label_community.setAlignment(Qt.AlignTop)
+        self.label_community.setOpenExternalLinks(True)
+        self.label_community.setTextInteractionFlags(Qt.TextBrowserInteraction)
+        self.label_community.setContentsMargins(15, 0, 25, 0)
+
+        self.label_legal.setWordWrap(True)
+        self.label_legal.setAlignment(Qt.AlignTop)
+        self.label_legal.setOpenExternalLinks(True)
+        self.label_legal.setTextInteractionFlags(Qt.TextBrowserInteraction)
+        self.label_legal.setContentsMargins(15, 0, 25, 0)
 
         icon_filename = "spyder_about"
         pixmap = QPixmap(get_image_path(icon_filename))
         self.label_pic = QLabel(self)
         self.label_pic.setPixmap(
-            pixmap.scaledToWidth(64, Qt.SmoothTransformation))
-        self.label_pic.setAlignment(Qt.AlignTop)
+            pixmap.scaledToWidth(100, Qt.SmoothTransformation))
+        self.label_pic.setAlignment(Qt.AlignBottom)
+        self.info = QLabel((
+                    """
+                    <div style='font-family: "{font_family}";
+                        font-size: {font_size}pt;
+                        font-weight: normal;
+                        '>
+                    <p>
+                    <b>Spyder IDE</b>
+                    <br>{spyder_ver}
+                    <br> {revision}
+                    <br> """).format(
+                spyder_ver=versions['spyder'],
+                revision=revlink,
+                font_family=font_family,
+                font_size=font_size))
```

Comment:
This has too much indentation. Please follow the same convention of the labels above and indent it to four spaces, like this

```python
self.info = QLabel((
    """
    <div style='font-family: "{font_family}";
    ...
```